### PR TITLE
#66 Balking Pattern

### PR DESCRIPTION
**Pull request by Rzeposlaw**
Pull request for Balking Pattern issue #66

### CODE REVIEW

File: balking/src/main/java/com/iluwatar/balking/App.java [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+ */
+package com.iluwatar.balking;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
+import java.util.concurrent.TimeUnit;
+
```
Comment (iluwatar) [inline balking/src/main/java/com/iluwatar/balking/App.java]: Please describe the pattern and how the example implements it here in the comments

File: balking/src/test/java/com/iluwatar/balking/AppTest.java [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ */
+package com.iluwatar.balking;
+
+public class AppTest {
+  @org.junit.Test
```
Comment (iluwatar) [inline balking/src/test/java/com/iluwatar/balking/AppTest.java]: Nitpicking, but here import org.junit.Test and @Test annotation would be better.

**Comment by iluwatar** [comment]
@Rzeposlaw any updates on this? The pull request is about to expire.

**Comment by Rzeposlaw** [comment]
Hello, I'm sorry, I was busy. Today I will send the updated version. :)

**Comment by Rzeposlaw** [comment]
@iluwatar I'm ready for another review. :)

**Comment by iluwatar** [comment]
@Rzeposlaw looks good, thank you for the pattern contribution :+1:

**Pull request merged**